test/cmd/legacy-script.sh

```diff
@@ -395,10 +394,10 @@ runTests() {
 
    if [[ -n "${WHAT-}" ]]; then
     for pkg in ${WHAT}
-    do 
+    do
       # running of kubeadm is captured in hack/make-targets/test-cmd.sh
-      if [[ "${pkg}" != "kubeadm" ]]; then 
-        record_command run_${pkg}_tests
+      if [[ "${pkg}" != "kubeadm" ]]; then
+        record_command run_"${pkg}"_tests
```

Comment:
nit: quote the whole string
```suggestion
        record_command "run_${pkg}_tests"
```

Reply:
Updated :-)